sdk/monitor/monitor-opentelemetry-exporter/src/generated/models/index.ts

```diff
@@ -274,7 +274,7 @@ export enum KnownDataPointType {
   /** Measurement */
   Measurement = "Measurement",
   /** Aggregation */
-  Aggregation = "Aggregation"
+  Aggregation = "Aggregation",
```

Comment:
Please remove changes to monitor-opentelemetry project as this will conflict with updates I'm doing here
https://github.com/Azure/azure-sdk-for-js/pull/28230